packages/xo-server/src/models/remote.js

```diff
@@ -15,6 +15,7 @@ export class Remotes extends Collection {
     const remotes = await super.get(properties)
     forEach(remotes, remote => {
       remote.enabled = remote.enabled === 'true'
+      remote.connected = remote.error === ''
```

Comment:
and enabled?

Reply:
That's not what I meant: connected does not simply mean there is no error, it should take the enabled status into account.